app/server/server.go

```diff
@@ -33,9 +33,14 @@ import (
 // InitAndRun initializes the server and runs it
 func InitAndRun(conf config.Reader, dataStore store.DataStore) error {
 	ctx := context.Background()
-	dataStore.LoadWorkflows(ctx)
 
+	if conf.GetBool(dconfig.SettingDebugLog) {
```

Comment:
Can't you simply write it as:
```
debug := conf.GetBool(dconfig.SettingDebugLog)
log.Setup(debug)
```
? You skip the if and (maybe) make codecov a bit happier.

Reply:
sure. done.